src/index.html

```diff
@@ -1,12 +1,354 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__logo-link">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <div class="nav__link">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </div>
+          </li>
+          <li  class="nav__list-item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__list-item">
+            <a
+            href="#"
+            class="nav__link"
+            data-qa="nav-hover"
```

Comment:
Pay attention on indents
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="nav-hover"
```